modules/api/src/Helpers/PostCodeSanitiser.cs

```diff
@@ -0,0 +1,12 @@
+using GpConnect.NationalDataSharingPortal.Api.Helpers.Interfaces;
+
+namespace GpConnect.NationalDataSharingPortal.Api.Helpers 
+{
+    public class PostCodeSanitiser: IPostCodeSanitiser
+    {
+        public string Sanitise(string input)
+        {
+            return input;
+        }
+    }
+}
```

Comment:
Please Remove this file, sorry this is my mistake

Reply:
Done.